### PR TITLE
feat: added wishlist page

### PR DESCRIPTION
**Pull request by GuptaEesh**
Added wishlist page.
- Fully responsive(upto 300px)
- 2 buttons - Move to cart(CTA) and Remove from wishlist.

ecomm4

### CODE REVIEW

**Review by shivampandey0 — APPROVED** [review]
Good use of CSS variables for responsiveness.
Good to merge.

File: wishlist/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+                    <button class="primary btn without-shadow sm">Move to Cart</button>
+                    <button class="secondary btn sm">Remove from wishlist</button>
+                </footer>
+            </div>
+        </section>
+    </div>
+</body>
+
+</html>
```
Comment (Ronak-Mutha) [inline wishlist/wishlist.html:109]: Just configure and run prettier once you're done with all your screens. Not required, just a suggestion.

**Review by Ronak-Mutha — COMMENTED** [review]
Good use of variables and best practices followed. Good to merge

File: wishlist/wishlist.html (hunk 1 above)
Comment (GuptaEesh) [reply wishlist/wishlist.html:109]: Yeah sure will do that. Thank you for the suggestion.

**Comment by GuptaEesh** [comment]
Got 2 reviews and merging the branch now.

**Pull request merged**